ios/Classes/CircleAnnotationMessager.swift

```diff
@@ -0,0 +1,601 @@
+// Autogenerated from Pigeon (v17.0.0), do not edit directly.
```

Comment:
There's a type in this file name. Should be CircleAnnotationMessenger

Reply:
Seems to be a bigger change as it will affect iOS, Android and Dart code(through codegen), I've created a ticket to tackle it separately https://mapbox.atlassian.net/browse/MAPSFLT-177